src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.time.LocalDate;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+
+public class DateValidators {
+
+    private DateValidators() { }
+
+    public static boolean validateDateIsInPast(LocalDate compareToDate, String location, Errors errs, String loggingContext) {
+        if (compareToDate.isAfter(LocalDate.now())) {
+            setErrorMsgToLocation(errs, location, location + ValidationMessages.DATE_NOT_IN_PAST_ERROR_MESSAGE);
+            ApiLogger.infoContext(loggingContext, location + " date should be in the past");
+            return false;
+        }
+
+        return true;
+    }
+
+    public static boolean validateDateIsWithinLast3Months(LocalDate compareToDate, String location, Errors errs, String loggingContext) {
+        var localDateNow = LocalDate.now();
+        if (compareToDate.isBefore(localDateNow) && compareToDate.isAfter(localDateNow.minusMonths(3))) {
```

Comment:
Implementation suggests that 'today' would cause 'false' to be returned but method name tells me that today should be accepted - 'validateDateIsWithinLast3Months'. Which is correct? Maybe method name needs tweaking...

Reply:
Not sure what to do about this.

Reply:
Hi Martin, any name suggestion?

Reply:
So first question is, do we need this now? Web has a `checkDateIsWithinLast3Months()` function but seems to only be used by the 'due diligence' checks, so not for the 'tell us about the overseas entity screen'.

`checkDateIsWithinLast3Months()` in web does seem to include 'today' though, so name of this function might be fine (if it remains), just implementation that needs changing - start with a unit test to prove my theory that implementation is wrong...

Reply:
Thanks Martin, you are right,  we do not need it for the Entity was part of the Utils task, that probably should have been in a different task instead of a subtask. 
The date utils will be moved in a separate task. Anyway waiting your unit test to prove me wrong :D

Reply:
Date implementation changed to include today by checking date is before tomorrow (today plus a day) rather than before today.